src/_utils/interfaces.ts

```diff
@@ -11,26 +11,38 @@ export interface CommonFieldsProps {
 export interface A11yProps {
   id?: string
   role?: string
+  title?: string
+  tabIndex?: number
   'aria-label'?: string
   'aria-labelledby'?: string
   'aria-describedby'?: string
   'aria-controls'?: string
 }
 
-type A11yKeys = keyof A11yProps
+type A11yKeysWithStringValue = keyof Omit<A11yProps, 'tabIndex'>
+type A11yKeysWithNumberValue = keyof Pick<A11yProps, 'tabIndex'>
 
-export function pickA11yProps<T extends A11yProps>(source: T): Pick<T, A11yKeys> {
-  const returnValue = {} as Pick<T, A11yKeys>
-  const keys: A11yKeys[] = [
+export function pickA11yProps<T extends A11yProps>(
+  source: T,
+): Pick<T, A11yKeysWithStringValue | A11yKeysWithNumberValue> {
+  const returnValue = {} as Pick<T, A11yKeysWithStringValue | A11yKeysWithNumberValue>
+  const keysWithStringValue: A11yKeysWithStringValue[] = [
     'id',
     'role',
+    'title',
     'aria-label',
     'aria-labelledby',
     'aria-describedby',
     'aria-controls',
   ]
-  keys.forEach(k => {
+  const keysWithNumberValue: A11yKeysWithNumberValue[] = ['tabIndex']
+
+  keysWithStringValue.forEach(k => {
+    returnValue[k] = source[k]
+  })
+  keysWithNumberValue.forEach(k => {
```

Comment:
`title` prop can't have a `number` value for example, so I split key props in two list to solve ts issue.

Reply:
Using something like `source.reduce()` might be a bit nicer

Reply:
Right it's cleaner now.